java/demo/src/main/resources/application.properties

```diff
@@ -0,0 +1,23 @@
+
+swagger.enable=true
+
+
+spring.redis.host=127.0.0.1
+#Redis��������Ӷ˿�
+spring.redis.port=6379
+#Redis������������루Ĭ��Ϊ�գ�
+spring.redis.password=
+#��ӳ���������ʹ�ø�ֵ��ʾû�����ƣ�
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 97f315fa-b57c-4b39-90ce-cfc5a8d3109a -->